### ISSUE TITLE
v1.4.0 does not support java8 

### ISSUE THREAD

**Issue by x-xoom** [ipfs-shipyard/java-ipfs-http-client#214]
v1.4.0 does not support java8

**Comment by kevodwyer** [comment]
That is correct. Java 8 was released 18th March 2014. Java 20 is released next month. Many projects are moving on from Java 8. According to wikipedia, support for Java 8 is ending soon (vendor dependent) : https://en.wikipedia.org/wiki/Java_version_history
